.github/workflows/docs.yml

```diff
@@ -37,13 +37,12 @@ jobs:
           python-version: "3.9"
       - name: Install dependencies
         run: |
+          sudo apt-get install -y graphviz
           python -m pip install --upgrade pip
           pip install -r requirements_dev.txt
       - name: Generate docs
         run: |
           make docs
-        env:
-          PYTHONPATH: "."
```

Comment:
add this in order to avoid problems in the future and for a consistent structure, also used in TRLC at this step in the CI